docs/reference/configuration/mbed_targets.md

```diff
@@ -192,7 +192,13 @@ Enabling `is_disk_virtual` adds delay after flashing firmware binary to make sur
 
 #### supported_toolchains
 
-The `supported_toolchains` property is the list of toolchains that support a target. The allowed values for `supported_toolchains` are `ARM`, `uARM`, `ARMC6`, `GCC_ARM` and `IAR`.
+The `supported_toolchains` property is the list of toolchains that support a target. The allowed values for `supported_toolchains` are `ARM`, `uARM`, `ARMC5`, `ARMC6`, `GCC_ARM` and `IAR`.
+
+If you specify `ARMC5` in `supported_toolchains`, it means the corresponding target supports `Arm Compiler 5.06 update 6`.
```

Comment:
We should be explicit that any new targets should support ARMC6 not 5.

Reply:
@bulislaw - Please see the updates.